lab-jonathan/model/guitars.js

```diff
@@ -0,0 +1,31 @@
+let mongoose = require('mongoose');
+let Schema = mongoose.Schema;
+let createError = require('http-errors');
+
+let Owner = require('./owners.js');
+
+let guitarSchema = Schema({
+  make: {type: String, required: true},
+  model: {type: String, required: true},
+  listID: [{type: Schema.Types.ObjectId, ref: 'owners'}], 
```

Comment:
Given that a guitar will only have one owner, this doesn't need to be an array value, rather just an object literal like the make and model.